### PR TITLE
Cleanup JavaScript page

### PR DESCRIPTION
**Pull request by joelfranusic-okta**
Resolves: OKTA-107492

### CODE REVIEW

**Comment by joelfranusic-okta** [comment]
Here is what the change to `code/javascript/index.html` looks like:

```sh
$ diff -u code/javascript/index.html _site/code/javascript/index.html  | tail -34
 	    <!-- START Subpages Grid -->
 	<div class="Row">
-      <!-- index.id &quot;/code/javascript/okta_auth_sdk&quot; -->
-	  <div class="Column--4 Column--small-12">
-        <a href="/code/javascript/okta_auth_sdk.html">
-	      <h3 class="h4">Okta Auth SDK</h3>
-	    </a>
-	    <p>
-	      Easily add sign-in capabilities to your fully branded web site.
-	    </p>
-	  </div>
-      <!-- index.id &quot;/code/javascript/okta_sign-in_widget&quot; -->
 	  <div class="Column--4 Column--small-12">
         <a href="/code/javascript/okta_sign-in_widget.html">
 	      <h3 class="h4">Okta Sign-In Widget</h3>
 	    </a>
 	    <p>
-	      Easily add sign-in capabilities with Okta's pre-packaged UI component.
+	      A drop-in widget with custom UI capabilities to power sign-in with Okta.
 	    </p>
 	  </div>
-      <!-- index.id &quot;/code/javascript/okta_sign-in_widget_ref&quot; -->
 	  <div class="Column--4 Column--small-12">
-        <a href="/code/javascript/okta_sign-in_widget_ref.html">
-	      <h3 class="h4">Okta Sign-In Widget Reference</h3>
+        <a href="/code/javascript/okta_auth_sdk.html">
+	      <h3 class="h4">Okta Auth SDK</h3>
 	    </a>
 	    <p>
-	      Reference information for customizing the Okta Sign-In Widget.
+	      A Javascript wrapper for Okta's Authentication APIs. Build your own branded UI and power sign-in with Okta.
 	    </p>
 	  </div>
 	</div>
```

**Comment by joelfranusic-okta** [comment]
@mystiberry-okta and @johngronberg-okta for your review.

**Comment by joelfranusic-okta** [comment]
Ping: @mystiberry-okta and @johngronberg-okta

**Comment by johngronberg-okta** [comment]
@joelfranusic-okta the code changes look fine, but could you add links to the top of okta_auth_sdk.md and okta_sign-in_widget.md that tell the reader where they can view the code on GitHub?

**Comment by mystiberry-okta** [comment]
@joelfranusic-okta  sorry for the delay. I don't understand what this is, or why we're hiding okta_sign-in_widget_ref.md, but the text updates look great.

**Comment by johngronberg-okta** [comment]
Users were getting confused by having multiple links to Okta Sign in Widget on http://developer.okta.com/code/javascript/. 

They should only go to the reference after reading the primer, so we simplified their options.

**Comment by mystiberry-okta** [comment]
It's not a great idea to completely hide a topic from the TOC, though, as internal and external customers who do want it (after reading the primer) won't be able to find it if their only path is via the primer. I mention this only because I answer multiple questions a month from internal customers about hidden topics. It's really not the best architecture. I don't think hiding it from this page is a good idea: http://developer.okta.com/code/javascript/

Can give them instruction in the excerpt instead of hiding this?

**Comment by johngronberg-okta** [comment]
Hm. I would propose we link to the reference and the Github repo from the overview portion of the okta_sign-in_widget.md page. Presenting too many similar sounding options to users causes too much confusion, and I'm not convinced keeping a link to the reference that is separate is the right approach.

**Comment by mystiberry-okta** [comment]
If it's not in the landing page, people who know it exists will be confused. That's a simple fact. Agree that if the differences between the topics aren't clear, we can fix that. Hiding things will cause a different problem, not wholly fix the problem we have.

File: _source/_code/javascript/okta_sign-in_widget.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 layout: docs_page
 title: Okta Sign-In Widget
-excerpt: Easily add sign-in capabilities with Okta's pre-packaged UI component.
+excerpt: A drop-in widget with custom UI capabilities to power sign-in with Okta.
```
Comment (johngronberg-okta) [inline _source/_code/javascript/okta_sign-in_widget.md]: Let's update this excerpt to: 

excerpt: A drop-in widget with custom UI capabilities to power sign-in with Okta. An <a href="/code/javascript/okta_sign-in_widget_ref.html">in-depth reference</a> is also available.

**Comment by mystiberry-okta** [comment]
Looks good!

**Comment by joelfranusic-okta** [comment]
@johngronberg-okta I updated the excerpt per your suggestion. I'll open a new PR for the links to GitHub repos in `okta_auth_sdk.md` and `okta_sign-in_widget.md`.

@mystiberry-okta - ready to merge pending approval from @johngronberg-okta

**Comment by johngronberg-okta** [comment]
lgtm

**Comment by johngronberg-okta** [comment]
btw- I see that github did it's auto complete on my example link- thanks for catching that @joelfranusic-okta

**Pull request closed without merging**